lib/parzival_web/live/landing/home_live/index.html.heex

```diff
@@ -301,6 +301,41 @@
   </div>
 </div>
 
+<div class="flex flex-row mt-10 sm:mt-20 lg:mt-48 min-h-[500px]">
+  <div class="absolute transform -rotate-90 translate-y-40 -translate-x-[100px] md:-translate-x-[100px]">
+    <span class="title">
+      Find us
+    </span>
+  </div>
+  <div class="flex relative flex-col gap-y-6 mt-3 ml-24 w-full lg:flex-row lg:ml-28 xl:ml-48">
+    <div class="mx-0 mt-8 w-full lg:mr-10 lg:mb-10 lg:ml-12">
+      <p class="text-sm font-bold tracking-wide text-center text-white sm:text-lg lg:text-left xl:tracking-widest">
+        How to find us
+      </p>
+      <p class="text-xs font-extralight text-center text-white lg:text-left tracking-[0.2em] sm:text-normal xl:tracking-[0.3em]">
+        The JOIN is free for participants and is organized by volunteers from CeSIUM, NECC, NEFUM, the Department of Software Engeneering, and the Scholl of Science from the university of Minho.
```

Comment:
There are typos: "Scholl" should be "School", "Engeneering" should be "Engineering". The Department is Department of Informatics, not just Software Engineering

Reply:
Thanks 🙏🙏🙏🙏